src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.ts

```diff
@@ -117,8 +117,8 @@ class TeamsReportDeviceUsageDistributionUserCountsCommand extends GraphCommand {
         return `${args.options.period} is not a valid period type. The supported values are D7|D30|D90|D180`;
       }
 
-      if (args.options.outputFilePath && !fs.existsSync(path.dirname(args.options.outputFilePath))) {
-        return 'Specified outputFilePath where to save the file does not exist';
+      if (args.options.outputFile && !fs.existsSync(path.dirname(args.options.outputFile))) {
+        return `The specified path ${args.options.outputFile} doesn't exist`;
```

Comment:
I'm afraid this error message will still be confusing. In the command options, we're asking the user to specify the path to the report file that we'll create. Before we execute the command, we check if the path (not the file) exists. If it doesn't, in the current state, we're showing the following error: `The specified path /path/to/file/report.md doesn't exist`, pointing to the file, while we're checking for the existance of the directory. It would be clearer if the error message said `The specified path /path/to/file/ doesn't exist`. That way the user would know that it's about the path rather than the file.

Reply:
Sure, This is fixed.